### ISSUE TITLE
Define component tag in CSS

### ISSUE THREAD

**Issue by ai** [kossnocorp/decss#20]
Is it a way to define `tag` in CSS? Like `base` from React CSS competent.

I think it will be nice to have it, since you will work it only once. Right now you can use your component in many places and it is easy to forget to set `tag` for `Button`.

**Comment by kossnocorp** [comment]
@ai hey, it's not possible at the moment. There is no way (or I didn't find it) to implement the feature without changing css-loader. See https://github.com/kossnocorp/decss/issues/17#issuecomment-385466941 for more details.

I'd love to have this, and in the future, I'll probably try to tackle it if no one else does it.

**Comment by ai** [comment]
Sad to hear :(.

BTW, I am planning to promote your awesome tool in my “PostCSS way” talk.

**Comment by kossnocorp** [comment]
@ai amazing 🤗 I appreciate it a lot! When and where it will happen?

**Comment by ai** [comment]
@kossnocorp PiterCSS this Friday

**Comment by kossnocorp** [comment]
@ai 👏
